src/panels/lovelace/cards/hui-vertical-stack-card.ts

```diff
@@ -0,0 +1,87 @@
+import { html, LitElement } from "@polymer/lit-element";
+
+import computeCardSize from "../common/compute-card-size.js";
+import createCardElement from "../common/create-card-element.js";
+
+import { LovelaceCard, LovelaceConfig } from "../types.ts";
+
+interface Config extends LovelaceConfig {
+  cards: LovelaceConfig[];
+}
+
+class HuiVerticalStackCard extends LitElement
+  implements LovelaceCard {
+  protected config: Config;
+  private mHass: HomeAssistant;
+
+  static get properties() {
+    return {
+      config: {},
+    };
+  }
+
+  public getCardSize() {
+    let totalSize = 0;
+    for (const element of this.shadowRoot.querySelectorAll('#root > *')) {
+      totalSize += computeCardSize(element);
+    };
+
+    return totalSize;
+  }
+
+  public setConfig(config: Config) {
+    if (!config || !config.cards || !Array.isArray(config.cards)) {
+      throw new Error("Card config incorrect");
+    }
+
+    this.config = config;
+  }
+
+  protected render() {
+    if (!this.config) {
+      return html``;
+    }
+
+    return html`
+      ${this.renderStyle()}
+      <div id="root">
+        ${this.config.cards.map(card => createCardElement(card, this.mHass))}
+      </div>
+    `;
+  }
+
+  private renderStyle() {
+    return html`
+      <style>
+        #root {
+          display: flex;
+          flex-direction: column;
+        }
+        #root > * {
+          margin: 4px 0 4px 0;
+        }
+        #root > *:first-child {
+          margin-top: 0;
+        }
+        #root > *:last-child {
+          margin-bottom: 0;
+        }
+      </style>
+    `;
+  }
+
+  set hass(hass: HomeAssistant) {
+    this.mHass = hass;
```

Comment:
`this._hass`

Reply:
TS did not like _hass as a variable name

Reply:
It allows it with Travis I believe. Not sure if thats a good or bad thing

Reply:
Disregard this is false. Entities Conversion failed because of this

Reply:
let's disable that rule in TS Lint

Reply:
`"variable-name": [true, "ban-keywords", "check-format", "allow-leading-underscore"]`

Does this look right?

Reply:
if the error stops happening: yeah 😁